### ISSUE TITLE
Get rid of run 32 and run 64 command line arguments

### ISSUE THREAD

**Issue by 0mp** [0mp/io-touchpad#154]
Some time ago @michal7352 suggested that we should get rid of `run 32` and `run 64` modes.

The idea is to mix hard-coded symbols with the user-defined ones. 

The system bitness is not a problem because it can be determined during the installation - Makefile should be able to tell if the system is 32-bit or 64-bit.

**Comment by michal7352** [comment]
Not exactly. I suggested to get rid of run user, and mix appropriate symbols together. Of course, if you have an idea how to join 32 and 64, it would be great. So there are two issues actually.
